### PR TITLE
Updated METBKA csv files

### PR DESCRIPTION
**Pull request by rtravis-whoi**
Updated CC_use_velpt from 0 to 1 (per discussion in data mtg on 11/7/19)
Also updated LGR # and renamed 3 files

### CODE REVIEW

**Comment by rtravis-whoi** [comment]
File for reference...
[METBK_Cal_Updates_2019-11-07.xlsx](https://github.com/WHOIGit/ooicgsn-asset-management/files/3821746/METBK_Cal_Updates_2019-11-07.xlsx)

File: calibration/METBKA/CGINS-METLGR-00015__20150507.csv [hunk 1]
```diff
@@ -0,0 +1,9 @@
+serial,name,value,notes
+LGR015,CC_depth_of_conductivity_and_temperature_measurements_m,1.11,CP01CNSM-00003
+LGR015,CC_height_of_air_humidity_measurement_m,4.255,
+LGR015,CC_height_of_air_temperature_measurement_m,4.255,
+LGR015,CC_height_of_windspeed_sensor_above_sealevel_m,4.74,
+LGR015,CC_jcool,1,Constant
+LGR015,CC_jwarm,1,Constant
+LGR015,CC_zinvpbl,600,Constant
+LGR015,CC_use_velpt,1,
```
Comment (snwhite) [inline calibration/METBKA/CGINS-METLGR-00015__20150507.csv]: Okay.  Replaces CGINS-METLGR-99992__20150507.csv.

File: calibration/METBKA/CGINS-METLGR-00014__20150507.csv [hunk 2]
```diff
@@ -0,0 +1,9 @@
+serial,name,value,notes
+LGR014,CC_depth_of_conductivity_and_temperature_measurements_m,0.98,CP01CNSM-00003 CP03ISSM-00002 CP04OSSM-00002
+LGR014,CC_height_of_air_humidity_measurement_m,4.255,
+LGR014,CC_height_of_air_temperature_measurement_m,4.255,
+LGR014,CC_height_of_windspeed_sensor_above_sealevel_m,4.74,
+LGR014,CC_jcool,1,Constant
+LGR014,CC_jwarm,1,Constant
+LGR014,CC_zinvpbl,600,Constant
+LGR014,CC_use_velpt,1,
```
Comment (snwhite) [inline calibration/METBKA/CGINS-METLGR-00014__20150507.csv]: This file only applies to CP01CNSM-00003.  The note on line 2 should be corrected.

The file CGINS-METLGR-99991__20150507.csv which was deleted, should remain for CP03ISSM-00002 and CP04OSSM-00002.

File: calibration/METBKA/CGINS-METLGR-00020__20151114.csv [hunk 3]
```diff
@@ -0,0 +1,9 @@
+serial,name,value,notes
+LGR020,CC_depth_of_conductivity_and_temperature_measurements_m,1.37,GA01SUMO-00002
+LGR020,CC_height_of_air_humidity_measurement_m,5.255,
+LGR020,CC_height_of_air_temperature_measurement_m,5.255,
+LGR020,CC_height_of_windspeed_sensor_above_sealevel_m,5.74,
+LGR020,CC_jcool,1,Constant
+LGR020,CC_jwarm,1,Constant
+LGR020,CC_zinvpbl,600,Constant
+LGR020,CC_use_velpt,1,
```
Comment (snwhite) [inline calibration/METBKA/CGINS-METLGR-00020__20151114.csv]: Oka.  This file replaces CGINS-METLGR-00025__20151114.csv.

File: calibration/METBKA/CGINS-METLGR-00014__20150507.csv (hunk 2 above)
Comment (rtravis-whoi) [reply calibration/METBKA/CGINS-METLGR-00014__20150507.csv]: Updated CGINS-METLGR-00014__20150507.csv per SW comment.  
Re-created file CGINS-METLGR-99991 per SW comment - however the date of the file is 20150509 (not 20150507) per WHOI asset tracking deploy date and listed in METBK_Cal_updates spreadsheet.  Both ISSM-2 AND OSSM-2 were deployed on 20150509.

Comment (snwhite) [reply calibration/METBKA/CGINS-METLGR-00014__20150507.csv]: This is good.

File: calibration/METBKA/CGINS-METLGR-99991__20150509.csv [hunk 4]
```diff
@@ -1,9 +1,9 @@
...
-UNKNOWN,CC_depth_of_conductivity_and_temperature_measurements_m,1.11,CP01CNSM-00003
+UNKNOWN,CC_depth_of_conductivity_and_temperature_measurements_m,0.98,CP03ISSM-00002 CP04OSSM-00002
 UNKNOWN,CC_height_of_air_humidity_measurement_m,4.255,
 UNKNOWN,CC_height_of_air_temperature_measurement_m,4.255,
 UNKNOWN,CC_height_of_windspeed_sensor_above_sealevel_m,4.74,
 UNKNOWN,CC_jcool,1,Constant
 UNKNOWN,CC_jwarm,1,Constant
 UNKNOWN,CC_zinvpbl,600,Constant
-UNKNOWN,CC_use_velpt,0,
+UNKNOWN,CC_use_velpt,1,
```
Comment (snwhite) [inline calibration/METBKA/CGINS-METLGR-99991__20150509.csv]: This is correct, but looks confusing.
Essentially CGINS-METLGR-99991__20150507 and CGINS-METLGR-99992__20150507 are deleted (go away). 
A new file is created called CGINS-METLGR-99991__20150509, which was essentially the CGINS-METLGR-99991__20150507 file with a new date and only applying to ISSM and OSSM.

**Pull request merged**